### ISSUE TITLE
Emoji are not visible using Linux

### ISSUE THREAD

**Issue by elamperti** [notwaldorf/mojibrag#17]
Emoji appears as squares for most of the characters. Can we discuss here how would be the best way to add a [twemojify!](https://github.com/notwaldorf/twemojify) button (or a similar feature)? 
It would enable Linux users to use mojibrag too! (probably applies to users of old Windows versions as well)

![messages](https://cloud.githubusercontent.com/assets/910672/19373703/a4d10f96-919c-11e6-8f41-f2b66abb41cb.jpg)

**Comment by notwaldorf** [comment]
That's a Linux problem, not the app's fault, unfortunately (it just uses your native emoji font to render). However, [Here](http://www.omgubuntu.co.uk/2016/08/enable-color-emoji-linux-google-chrome-noto)'s how to fix that!

**Comment by notwaldorf** [comment]
Actually, I'll reopen and rename this issue as a feature request, since a Twemojify button isn't a bad idea

**Comment by anxolerd** [comment]
What about not adding a twemojify button, but using an emoji-powered font on site?

**Comment by wongmjane** [comment]
I am using Chromium on a Linux distro with [Noto Color Emoji](https://www.google.com/get/noto/help/emoji/) and it works.

![screenshot from 2016-10-17 16-35-42](https://cloud.githubusercontent.com/assets/1332975/19454595/cbfd3fac-9487-11e6-8f77-390bbbd4c7e4.png)

**Comment by anxolerd** [comment]
@wongmjane the same here, but there are a lot of guys who do not use Noto Color Emoji font and see that ugly squares.

**Comment by notwaldorf** [comment]
@anxolerd Yes, but that also means you don't see any emoji ever, on any web page :(

**Comment by anxolerd** [comment]
@notwaldorf: Is there any good way to insert html into Polymer template? And is there any way to escape html string outside the Polymer template? What I want to do is:

```js
<template>{prepare(someData)}</template>
...
function prepare(data) {
    var escaped = escapeHtml(data);
    return wrapIntoHtml(escaped);
}
```

That might work pretty well with `twemoji` to display emoji on page.

**Comment by elamperti** [comment]
@anxolerd hey, I have the same problem trying to change the messages, we should solve it safely to avoid XSS.

 I uploaded my progress [in this branch](https://github.com/elamperti/mojibrag/tree/twemojify), hope it helps you or someone else working on this too.

Other stuff related to this feature:
  * Each emoji in the **preview area has a `<select>`** with emoji options (and -surprise!- it doesn't handle images well), so it should be changed to something better (maybe [iron-dropdown](https://elements.polymer-project.org/elements/iron-dropdown)?)
  * Once we have twemoji added, **there are many elements that should be parsed** (page title, lang switcher, channel names, the feed, preview area). Instead of binding everything we may run it once for each element when twemoji loads.
